src/arrayMethodSort.js

```diff
@@ -3,10 +3,35 @@
 /**
  * Implement method Sort
  */
+
+const compareStringFunction = (x, y) => {
+  const stringX = x.toString();
+  const stringY = y.toString();
+
+  return stringX > stringY ? 1 : stringX < stringY ? -1 : 0;
```

Comment:
better to use `localeCompare`

Reply:
just commit called `the proposed option does not work`, it is not suitable. Describe the code you wrote, you can also always write to the chat to solve the problem and find out why the code does not work